src/web/content/static/css/letterpress.css

```diff
@@ -367,17 +367,17 @@ a.sms-large-font:hover {
 
 /* Home ******************* */
 .sms-home-main-container {
-    height: 80vh;
+    height: 85vh;
     display: flex;
     flex-direction: column;
 }
 .sms-home-header {
-    height: 20vh;
```

Comment:
The header was too tall. This is the one with the logo and menu items, the navigation header.